Unit testing contracts

mjaric opened the pull request.

mjaric committed:
testing hearts trick

mjaric committed:
unit tests for JackOfClubs

mjaric committed:
maximum uni test

mjaric committed:
minimum test cases

mjaric committed:
reformat and trace unit test in pipeline

mjaric committed:
Queens

mjaric committed:
lora, king hearts last trick

mjaric committed:
implementing missing presence module

mjaric committed:
adding excoveralls

mjaric committed:
using builtin coverage tool

mjaric committed:
fixing format

mjaric committed:
fixing missing CI authorization

mjaric committed:
significant refactoring on contracts, removing legacy module resolution

mjaric committed:
disabling build for a moment

mjaric committed:
disabling build for a moment

mjaric committed:
removing build phase for now

mjaric committed:
fixing upload artifacts

mjaric committed:
covering more modules and lora app pass 83% of coverage

mjaric committed:
foramtting code

burmajam commented on apps/lora_web/lib/lora_web/live/game_live.ex.bak:
Is this here by mistake?

mjaric replied:
commited by mistake. Due syntax error it made a copy, wrote generated file again and left back (didn't deleted)

burmajam reviewed (approved):
tl;dr

mjaric committed:
more test

mjaric committed:
fixing unit tests

mjaric committed:
fixing unit tests

mjaric committed:
removing bak file

mjaric committed:
redesign

mjaric committed:
better ux

mjaric committed:
cleanup

mjaric committed:
dumb issue with copilot

vs code become slow and some changes were not persisted on disk

updated to vs code 100.2, will see if issue persists
